frontend/packages/dev-console/src/components/formik-fields/field-types.ts

```diff
@@ -1,5 +1,6 @@
 export interface InputFieldProps {
   type?: string;
+  'data-test'?: string;
```

Comment:
Do not include `data-` attributes in prop types.

Reply:
When I first added it, I thought it gave me an error. No error now, so I guess I can remove it.